documents/Libraries/stdlib/sx-osl/mx_modulo_color3FA.inline

```diff
@@ -0,0 +1 @@
+fmod({{in1}}, color({{in2}},{{in2}},{{in2}}))
```

Comment:
Use specific color3 and vector3 packed using input float to avoid testrender support issue.